package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@github/auto-complete-element",
-  "version": "3.0.2",
+  "version": "3.1.0",
```

No need to bump the version, we will do that when we cut a release
```suggestion
  "version": "3.0.2",
```

---

Because I added the "autoselect" feature, it seemed worth a minor bump? Happy to switch back if you disagree -- just wanted to mention the reasoning.

---

I agree it's worth a minor bump 😄.  Just doesn't need to be changed within this PR.  We will do a minor bump when we actually release it.